bats/tests/registry/creds.bats

```diff
@@ -103,6 +108,22 @@ EOF
     wait_for_container_engine
 }
 
+@test 'verify credential is set correctly' {
+    verify_default_credStore
+}
+
+verify_default_credStore() {
+    local CREDHELPER_NAME="$(basename "$CRED_HELPER" .ext | sed s/docker-credential-//)"
+
+    assert_file_exist "$CONFIG_FILE"
+    if command -v jq; then
```

Comment:
`jq` is a prerequisite for our BATS tests; we don't need to provide an alternate implementation.

Having a test perform different operations based on the presence/absence of other tools on the system is a potential source of error: maybe the 2 branches are not covering the same functionality?

Reply:
Good to know `jq` is a pre-rec